### DISCUSSION TITLE
VSCode Terminal: transparent not working

### DISCUSSION THREAD

**Discussion by floh96** [JanDeDobbeleer/oh-my-posh#182] [Troubleshoot] [answered]
### Prerequisites

- [x ] I have read and understand the `CONTRIBUTING` guide
- [x ] I looked for duplicate issues before submitting this one

### Description

the transparent effekt is not working in the vscode terminal. It is displaying black background see screenshoots.

### Environment

- Oh my Posh version: 3.44.1-beta
- Theme: fish with some [changes](https://gist.github.com/floh96/1f26bff98d6b87255146b98f148860ff)
- Operating System: windows 10
- Shell: powershell version 7.1.0
- Terminal: vscode integrated terminal

### Steps to Reproduce

open vscode integrated terminal

**Expected behavior:**

transparent effekt displays korrekt like in windows terminal 
![terminal](https://user-images.githubusercontent.com/49693964/99911635-1cb2f800-2cf6-11eb-8f14-76b308127d09.PNG)

**Actual behavior:**

transparent effekt displays not korrekt (black)
![vscode](https://user-images.githubusercontent.com/49693964/99911527-5fc09b80-2cf5-11eb-92da-fef9b1ce4e38.PNG)

**Comment by JanDeDobbeleer** [comment]
That's a bug in VSCode not handling the inverted ANSI correctly. Can you open an issue there?

**Comment by floh96** [comment]
@JanDeDobbeleer sorry to bother you could you open the issue i think you could it explain better :)

**Comment by JanDeDobbeleer** [comment]
That's a good idea. I'll do it tomorrow with a proper sample they can use to reproduce this.

**Comment by JanDeDobbeleer** [comment, marked as answer]
See https://github.com/microsoft/vscode/issues/111762 for more context

**Comment by aessing** [comment]
Hi,

I have the same issue in the Windows Terminal App from the Microsoft Store.

Windows Terminal - Version: 1.4.3243.0 (from Microsoft Store)
Powershell - 7.1.0 (from Microsoft Store)
Windows 10 - 19042.685

Did you experienced something similar in the Terminal App with Acrylic Effect set to on?

Cheers

Andre

**Reply by floh96** [reply]
Yeah when I'm setting `"useAcrylic": true,` the transparent effekt is not working like in vscode.
Could you file an issue in the terminal repository? 
Since vscode is using some things from the terminal repository for it's terminal maybe it's a terminal problem?

**Reply by aessing** [reply]
@floh96 Sure, I can do... but I'm not an expert on terminal and coding. Could you give me a hint which function is not working in terminal?
Is it the same as in VSCode? Or should it use some transparency acrylic effect it doesn't?

**Reply by floh96** [reply]
@aessing I'm also no expert 😄 I guess it's the same issue as in  microsoft/vscode#111762 maybe @JanDeDobbeleer can help?

**Reply by JanDeDobbeleer** [reply]
There’s another issue I’ve been meaning to look at which has this exact context. See #250.
